JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.owl

```diff
@@ -411,7 +411,7 @@
 
     <owl:DatatypeProperty rdf:about="http://landregistry.data.gov.uk/def/ppi/transactionDate">
         <rdfs:domain rdf:resource="http://landregistry.data.gov.uk/def/ppi/TransactionRecord"/>
-        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#date"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
```

Comment:
Please fix, this range needs to be `xsd:date`

Reply:
Fixed. This seems to be casued by TBoxGeneration. Even though the range was set as date in the csv file, TBoxGeneration generated range as string in the owl file.

Reply:
I know, this is a known bug of the TBox generator